samples/nrf9160/modem_shell/README.rst

```diff
@@ -720,6 +784,18 @@ The LEDs have the following functions:
 LED 3 (nRF9160 DK)/Blue LED (Thingy:91):
    Indicates the LTE registration status.
 
+Power measurements
+==================
+
+Power measurements can be conducted using `Power Profiler Kit II (PPK2)`_.
+To achieve proper power measurement numbers, UART must be disabled.
```

Comment:
Should we also add a link to PPK2 documentation, which describes how a DK should be connected for power measurements? This page describes how to setup PPK2 and DK for power measurement:

https://infocenter.nordicsemi.com/topic/ug_ppk2/UG/ppk/setting_up.html

Reply:
Yes, it is. If we don't want to add a second link here, I think we should at least add a sentence saying that see the PPK2 documentation for instructions how to setup the DK for power measurements. The documentation shows how to connect the wires etc. for both source meter and ampere meter modes.

Reply:
I've added your suggested changes almost as is. Please check.